### ISSUE TITLE
oh my zsh prompt is mistaken as input 

### ISSUE THREAD

**Issue by ryan-ju** [warpdotdev/Warp#948]
### Describe the bug

```text
For oh my zsh, the command prompts are misinterpreted as input, causing "command not found" error.
```

### To Reproduce

Steps to reproduce:
1. Install oh-my-zsh
2. Open Warp

### Expected behaviour

```text
oh my zsh prompt should be correctly escaped and printed
```

### Screenshots

<img width="841" alt="image" src="https://user-images.githubusercontent.com/1434997/162255015-78ed88c2-41b6-41a5-be79-c212a079ac4a.png">

**Comment by davidklemme** [comment]
I second that. Its unfortunate, since I will not change my current setup to test. Especially with the telemetry stuff (while in beta I guess)

**Comment by elviskahoro** [comment]
Hey!

tldr; We aren't able to parse your prompt! 

We have a prompt compatibility table in our docs!
Link to the documentation for enabling custom prompt / honoring PS1: https://docs.warp.dev/features/prompt​

I personally recommend using (starship)!

You can disable specific parts of your RC file just for Warp!
https://docs.warp.dev/help/known-issues#configuring-and-debugging-your-rc-files

```sh
if [[ $TERM_PROGRAM != "WarpTerminal" ]]; then
##### WHAT YOU WANT TO DISABLE FOR WARP - BELOW

    # POWERLEVEL10K
    # OH-MY-ZSH-THEMES
    # OH-MY-ZSH-PLUGINS
    # FIG
    # FZF
    # BIND keys like:
    # bindkey "^j" down-line-or-beginning-search
    # [[ -r "/usr/local/etc/profile.d/bash_completion.sh" ]] && 
    # . "/usr/local/etc/profile.d/bash_completion.sh"
    # eval "$(rbenv init -)"

##### WHAT YOU WANT TO DISABLE FOR WARP - ABOVE
fi
```

**Comment by warpdotdev-dx** [comment]
Hey!


tldr; We aren't able to parse your prompt! 


We have a prompt compatibility table in our docs!
Link to the documentation for enabling custom prompt / honoring PS1: https://docs.warp.dev/features/prompt​


I personally recommend using (starship)!


You can disable specific parts of your RC file just for Warp!
https://docs.warp.dev/help/known-issues#configuring-and-debugging-your-rc-files


```sh
if [[ $TERM_PROGRAM != "WarpTerminal" ]]; then
##### WHAT YOU WANT TO DISABLE FOR WARP - BELOW


# POWERLEVEL10K
# OH-MY-ZSH-THEMES
# OH-MY-ZSH-PLUGINS
# FIG
# FZF
# BIND keys like:
# bindkey "^j" down-line-or-beginning-search
# [[ -r "/usr/local/etc/profile.d/bash_completion.sh" ]] && 
# . "/usr/local/etc/profile.d/bash_completion.sh"
# eval "$(rbenv init -)"


##### WHAT YOU WANT TO DISABLE FOR WARP - ABOVE
fi
```


Will close this, btw!
